packages/dev-server-esbuild/test/ts.test.ts

```diff
@@ -84,7 +87,7 @@ class Bar {
         'application/javascript; charset=utf-8',
       );
       expectIncludes(text, '__decorate');
-      expectIncludes(text, 'this.x = "y";');
+      expectIncludes(text, '__publicField(this, "x", "y");')
```

Comment:
It seems like how ESBuild outputs this line has changed between versions.

Reply:
Before (0.17.19): https://esbuild.github.io/try/#dAAwLjE3LjE5AC0tbG9hZGVyPXRzIC0tdHNjb25maWctcmF3PSd7CiAgImNvbXBpbGVyT3B0aW9ucyI6IHsKICAgICJleHBlcmltZW50YWxEZWNvcmF0b3JzIjogdHJ1ZQogIH0KfQonIC0tdGFyZ2V0PSJjaHJvbWU2NCxlZGdlNzksZmlyZWZveDY3LHNhZmFyaTExLjEiAEBmb28KY2xhc3MgQmFyIHsKICBAcHJvcAogIHggPSAneSc7Cn0

After (0.19.2): https://esbuild.github.io/try/#dAAwLjE5LjIALS1sb2FkZXI9dHMgLS10c2NvbmZpZy1yYXc9J3sKICAiY29tcGlsZXJPcHRpb25zIjogewogICAgImV4cGVyaW1lbnRhbERlY29yYXRvcnMiOiB0cnVlCiAgfQp9CicgLS10YXJnZXQ9ImNocm9tZTY0LGVkZ2U3OSxmaXJlZm94Njcsc2FmYXJpMTEuMSIAQGZvbwpjbGFzcyBCYXIgewogIEBwcm9wCiAgeCA9ICd5JzsKfQ

Difference:

```diff
2a3
> var __defNormalProp = (obj, key, value) => key in obj ? __defProp(obj, key, { enumerable: true, configurable: true, writable: true, value }) : obj[key] = value;
11a13,16
> var __publicField = (obj, key, value) => {
>   __defNormalProp(obj, typeof key !== "symbol" ? key + "" : key, value);
>   return value;
> };
14c19
<     this.x = "y";
---
>     __publicField(this, "x", "y");
```